policy/lib/bundles_test.rego

```diff
@@ -133,94 +133,6 @@ test_missing_required_data if {
 	lib.assert_equal(bundles.missing_task_bundles_data, true)
 }
 
-test_newer_in_effect_version_exists_not_using_tags_newest if {
```

Comment:
These tests were removed because it tests an "internal" function that no longer exists. I believe there is sufficient coverage elsewhere for the intended behavior.